saemref/install.sls

```diff
@@ -31,6 +31,29 @@ create-saemref-user:
   user.present:
     - name: {{ saemref.instance.user }}
 
+{% if saemref.install.dev %}
+dev dependencies:
+  pkg.installed:
+    - pkgs:
+      - python-pip
+      - python-virtualenv
+      - mercurial
+  pip.installed:
+    - name: setuptools
+    - ignore_installed: true
+
+install saem_ref from hg:
+  pip.installed:
+    - name: hg+http://hg.logilab.org/master/cubes/saem_ref#egg=cubicweb-saem_ref
+    - user: {{ saemref.instance.user }}
+    - install_options:
+      - --prefix=/home/{{ saemref.instance.user }}/local
```

Comment:
Forget about this, I just see the code below and it better to have an explicit separated installation path here.